tests/fs/test_memfs.py

```diff
@@ -1,212 +0,0 @@
-from os import fspath
```

Comment:
We never used this version of memfs directly. This has been added to `morefs` project and is tested there in case we need in the future.